### PR TITLE
Feature/shallow copies of images in layers

### PR DESCRIPTION
**Pull request by adreyfus**
Use shallow copies of images in layers, to prevent the function `convertImageToFalseColorImage()` from altering original images.

### CODE REVIEW

**Comment by swederik** [comment]
Hmm this is indeed a problem we had overlooked. Thanks for the PR.

File: src/stackTools/fusionRenderer.js [hunk 1]
```diff
@@ -31,9 +31,9 @@ export default class FusionRenderer {
         const currentLayerId = this.layerIds[0];
         const layer = cornerstone.getLayer(element, currentLayerId);
 
-        layer.image = image;
+        layer.image = $.extend({}, image);
```
Comment (swederik) [inline src/stackTools/fusionRenderer.js]: Could we do this without jQuery instead? Given that we still eventually plan to remove it.

Comment (adreyfus) [reply src/stackTools/fusionRenderer.js]: The ES6 equivalent is Object.assign(), but not IE compatible (and honestly I don't think it's a good idea to remove jQuery).

Comment (adreyfus) [reply src/stackTools/fusionRenderer.js]: I didn't see there is already many Object.assign() in the code.

Comment (adreyfus) [reply src/stackTools/fusionRenderer.js]: done :)

**Pull request merged**